test/hastings_delete_db_test.erl

```diff
@@ -84,7 +84,7 @@ should_delete_index_after_deleting_database(DbName) ->
         GeoDirExistsBefore = filelib:is_dir(GeoIdxDir),
         
         fabric:delete_db(DbName, [?ADMIN_CTX]),
-        meck:wait(hastings_vacuum, clean_shard_db, '_', 5000),
+        meck:wait(8, hastings_index, destroy, '_', 5000),
```

Comment:
Can you explain the `8` here? If it's the q value, should we use `config` instead?

Reply:
Yes, it is q. Good suggestion, and I just added new commit to get q from configuration. Thanks